content/en/blog/2018/soft-multitenancy/index.md

```diff
@@ -28,7 +28,7 @@ instance. Separation between the tenants is provided by Kubernetes namespaces an
 One use case for this deployment model is a shared corporate infrastructure where malicious
 actions are not expected, but a clean separation of the tenants is still required.
 
-Potential future Istio multi-tenant deployment models are described at the bottom of this
+Potential future of Istio multi-tenant deployment models are described at the bottom of this
```

Comment:
I think the original text is correct. Below we talk about future models, not the future **of the** models. These are very close in definition, but I think the later implies more of a keep/remove of the models as a whole while the former is more of how the models might change or additional models added. But, this could just be my interpretation.